src/app/Scenes/Tag/TagArtworks.tsx

```diff
@@ -24,16 +24,18 @@ interface TagArtworksProps extends TagArtworksContainerProps {
 
 export const TagArtworks: React.FC<TagArtworksProps> = ({ tag, relay, openFilterModal }) => {
   const tracking = useTracking()
-  const artworksTotal = tag.artworks?.counts?.total ?? 0
+  const artworksTotal = tag?.artworks?.counts?.total ?? 0
   const initialArtworksTotal = useRef(artworksTotal)
 
   const trackClear = () => {
-    tracking.trackEvent(tracks.clearFilters(tag.id, tag.slug))
+    if (!!tag && !!tag.id && !!tag.slug) {
```

Comment:
Maybe easier to read 🤷 

```suggestion
    if (tag?.id && tag?.slug) {
```